docs/releases/1.9.2.txt

```diff
@@ -71,3 +71,6 @@ Bugfixes
 
 * Fixed a crash when using a reverse ``OneToOneField`` in
   ``ModelAdmin.readonly_fields`` (:ticket:`26060`).
+
+* Fixed a crash when using the ``migrate`` command with the ``--run-syncdb``
+  option (:ticket:`26135`).
```

Comment:
You could briefly describe the circumstances under which it happens -- right now it seems like it's completely broken which I think isn't the case.